pom.xml

```diff
@@ -73,7 +73,18 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
```

Comment:
@janosmeszaros: Please keep the `org.springframework` dependencies in one group.